streaming/base/converters/README.md

```diff
@@ -0,0 +1,5 @@
+### [Spark Dataframe Conversion](https://github.com/mosaicml/streaming/blob/main/streaming/base/converters/dataframe_to_mds.py)
```

Comment:
nit: you can remove the link and instead add it in the below line where you are mentioned `dataframeToMDS` method, just link the definition path there.